flake.nix

```diff
@@ -37,7 +61,8 @@
     devShell = pkgs.mkShell {
       buildInputs = with pkgs; [
         python3
-        nix
```

Comment:
Removed Nix here since `nixpkgs-hammer` will already have it in `PATH` when ran from the package (as tests do), and for development one will have Nix anyway. And stable Nix here would make updating flake annoying.